packages/twenty-front/src/modules/object-record/record-table/components/RecordTableBody.tsx

```diff
@@ -19,8 +19,8 @@ export const RecordTableBody = () => {
             </RowIndexContext.Provider>
           </RowIdContext.Provider>
         ))}
+        <RecordTableBodyFetchMoreLoader />
       </tbody>
-      <RecordTableBodyFetchMoreLoader />
```

Comment:
this should stay here as RecordTableBodyFetchMoreLoader is a <tbody> (otherwise, it will cause issue with infinite scroll and fetch more logic with a large number of records